apollo-router/src/plugins/include_subgraph_errors.rs

```diff
@@ -52,13 +52,26 @@ impl Plugin for IncludeSubgraphErrors {
         // Search for subgraph in our configured subgraph map.
         // If we can't find it, use the "all" value
         if !*self.config.subgraphs.get(name).unwrap_or(&self.config.all) {
+            let sub_name_response = name.to_string();
+            let sub_name_error = name.to_string();
             return service
                 .map_response(move |mut response: SubgraphResponse| {
                     if !response.response.body().errors.is_empty() {
+                        tracing::info!("redacted subgraph({sub_name_response}) errors");
```

Comment:
maybe a `tracing::error` for those messages, since they're about error management?

Reply:
I did originally have this as `warn`, but I changed it to `info` when I considered that redaction should be the expected norm (i.e.: information) and therefore I don't want to be causing concern to people. I'm fairly sure it's not an `error`.

Reply:
Personally I think this should be debug.
It's not an error to redact, and logging that something got redacted on every request feels like overkill.
Maybe inside the `activate` function something can be logged at info to indicate what redaction is happening going forward.

Reply:
right

Reply:
Fair enough